docs/src/main/asciidoc/security-openid-connect-multitenancy.adoc

```diff
@@ -209,8 +210,10 @@ Otherwise, it initiates an authorization code flow when authentication is requir
 [source,properties]
 ----
 # Default tenant configuration
-%prod.quarkus.oidc.auth-server-url=http://localhost:8180/realms/quarkus
+%prod.keycloak.url=http://localhost:8180
```

Comment:
Why can't a single property be used ? It looks more complex now

Reply:
I change it to be cloaser to quickstart but when I think about it let's leave it as `%prod.quarkus.oidc.auth-server-url=http://localhost:8180/realms/quarkus`